code/modules/mechs/premade/heavy.dm

```diff
@@ -56,7 +48,7 @@
 	exosuit_desc_string = "super-heavy reinforced manipulators"
 	icon_state = "heavy_arms"
 	desc = "Designed for durability and dishing out beatings, this heavy set of manipulators can both take and dish out beatings."
-	melee_damage = 25
+	melee_damage = 60 // You know , these things walk like a snail . Why would you even get close to this.
```

Comment:
I'm gonna put heavy arms on a fast mech and punt heads off of people.

Reply:
Sugg: Maybe disable strafing on treads, because strafing basically negates the balance of turn_delay?

Reply:
You need atleast 80 damage to one-tap a head, if 60 could actually one-tap valkrie would wreck.